--- post/cvm-initramfs.bb/cvm-initramfs.bb.old	2024-10-22 16:51:14.920404557 +0200
+++ post/cvm-initramfs.bb/cvm-initramfs.bb.new	2024-10-22 22:54:31.022695822 +0200
@@ -7,7 +7,7 @@
 
 CVM_DEPS = "busybox-mdev init-ifupdown initscripts base-files base-passwd netbase busybox-udhcpd"
 
-PACKAGE_INSTALL = "ca-certificates sysvinit busybox-udhcpd date-sync logrotate cronie ${CVM_DEPS} ${VIRTUAL-RUNTIME_base-utils} ${ROOTFS_BOOTSTRAP_INSTALL}"
+PACKAGE_INSTALL = "ca-certificates sysvinit busybox-udhcpd date-sync logrotate cronie nethermind lighthouse tdx-attestor ${CVM_DEPS} ${VIRTUAL-RUNTIME_base-utils} ${ROOTFS_BOOTSTRAP_INSTALL}"
 
 INITRAMFS_MAXSIZE = "20000000"
 
