--- post/cvm-initramfs.bb/cvm-initramfs.bb.old	2024-10-04 16:17:58.903093751 +0200
+++ post/cvm-initramfs.bb/cvm-initramfs.bb.new	2024-10-21 15:22:20.148251475 +0200
@@ -7,7 +7,7 @@
 
 CVM_DEPS = "busybox-mdev init-ifupdown initscripts base-files base-passwd netbase busybox-udhcpd"
 
-PACKAGE_INSTALL = "ca-certificates sysvinit busybox-udhcpd date-sync logrotate cronie ${CVM_DEPS} ${VIRTUAL-RUNTIME_base-utils} ${ROOTFS_BOOTSTRAP_INSTALL}"
+PACKAGE_INSTALL = "ca-certificates sysvinit busybox-udhcpd date-sync logrotate cronie nethermind lighthouse json-config ${CVM_DEPS} ${VIRTUAL-RUNTIME_base-utils} ${ROOTFS_BOOTSTRAP_INSTALL}"
 
 INITRAMFS_MAXSIZE = "20000000"
 
