--- post/setup/setup.old	2024-09-04 02:06:00.831520031 +0200
+++ post/setup/setup.new	2024-10-21 23:21:30.397972901 +0200
@@ -28,15 +28,20 @@
 verbose_output "Sourced the oe-init-build-env script"
 
 # Add the  meta-evm, meta-confidential-compute, meta-secure-core/meta-tpm2, meta-openembedded/meta-python, meta-openembedded/meta-oe layers meta-rust-bin meta-clang to bblayers.conf
-bitbake-layers add-layer ../meta-evm
+# bitbake-layers add-layer ../meta-evm
 bitbake-layers add-layer ../meta-confidential-compute
 bitbake-layers add-layer ../meta-openembedded/meta-oe
 bitbake-layers add-layer ../meta-openembedded/meta-python
 bitbake-layers add-layer ../meta-secure-core/meta-tpm2
-bitbake-layers add-layer ../meta-rust-bin
-bitbake-layers add-layer ../meta-clang
+# bitbake-layers add-layer ../meta-rust-bin
+# bitbake-layers add-layer ../meta-clang
+bitbake-layers add-layer ../meta-dotnet
+bitbake-layers add-layer ../meta-nethermind
+bitbake-layers add-layer ../meta-lighthouse-bin
+bitbake-layers add-layer ../meta-json-config
+bitbake-layers add-layer ../meta-attestation
 
-verbose_output "Added the meta-evm and meta-confidential-compute layers to bblayers.conf"
+verbose_output "Added the layers to bblayers.conf"
 
 # Return to the original directory
 popd
@@ -49,6 +54,15 @@
     verbose_output "Processing patch file $patch"
 
     # Apply the patch to the corresponding file in srcs/poky/build/conf/
-    patch -N $CURRENT_PATH/srcs/poky/build/conf/$filename -i $patch
+    # Ignore errors if the patch has already been applied
+    patch -N -f -s $CURRENT_PATH/srcs/poky/build/conf/$filename -i $patch 2>/dev/null || true
     verbose_output "Applied patch to $CURRENT_PATH/srcs/poky/build/conf/$filename"
 done
+
+sed -i 's/MACHINE ?= "tdx"/MACHINE ?= "tdx-gcp"/' $CURRENT_PATH/srcs/poky/build/conf/local.conf
+if ! grep -q 'INITRAMFS_MAXSIZE = "700000"' $CURRENT_PATH/srcs/poky/build/conf/local.conf; then
+    echo 'INITRAMFS_MAXSIZE = "700000"' >> $CURRENT_PATH/srcs/poky/build/conf/local.conf
+fi
+if ! grep -q 'DISTRO_FEATURES:append = " tpm2"' $CURRENT_PATH/srcs/poky/build/conf/local.conf; then
+    echo 'DISTRO_FEATURES:append = " tpm2"' >> $CURRENT_PATH/srcs/poky/build/conf/local.conf
+fi
